app/src/main/resources/db/migration/V1.1__catfacts.sql

```diff
@@ -0,0 +1,21 @@
+-- TABLE
+CREATE TABLE T_CATFACTS
+(
+  ID            NUMBER        NOT NULL,
+  FACT          VARCHAR2(255) NOT NULL,
+  LENGTH        NUMBER        NULL,
+  CREATED_DATE  TIMESTAMP     NOT NULL,
+
+  CONSTRAINT T_CATFACTS_PK PRIMARY KEY (ID)
+);
+
+-- SEQUENCE
+CREATE SEQUENCE  T_CATFACTS_SEQ
```

Comment:
Mulig dette er litt vel next-level?

Reply:
Kan hende. Trenger ikke gå i dybden på det, men det er vel noe de kommer til å møte i prosjeketene, så kan vel ha det i prosjektet?

Reply:
Ja, de kommer jo garantert til å møte mye større databaser og mer komplekse flyway-filer senere, men vi må nesten sette et scope for gjennomgangen. Dersom man aldri har sett en applikasjon før tenker jeg kanskje at dette er litt vel avansert å fokusere på.  Men for de med litt erfaring er det jo nice! :)